composer.json

```diff
@@ -4,8 +4,8 @@
   "keywords": ["laravel", "javascript", "language", "translations", "routes"],
   "type": "library",
   "require": {
-    "php": ">=5.5.9",
-    "illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*|5.5.*"
+    "php": ">=7.1",
+    "illuminate/support": "^5.1"
```

Comment:
Not sure if we also want to do v6/7/8/9 while we are here?

Reply:
Like this branch Adrien already made https://github.com/Pod-Point/laravel-javascript-lang/pull/1/files

Reply:
I'd rather do it step by step. I don't want to say it's gonna work with 9.x if won't be tested with this version. Right now we are upgrading Laravel to 5.8, so intentionally prevented it to be installed with 6.x and above.